library/docs/changelog.md

```diff
@@ -9,6 +9,30 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Upgrade Wire to `4.7.0`.
 - Upgrade Kotlin to `1.8.21`.
+- Upgrade Gradle to `8.1.1`.
+- Upgrade Kotest to `5.6.2`.
```

Comment:
This change doesn't affect library consumers and shouldn't be added to the changelog.